instance/management/commands/instance_statistics_csv.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2020 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app - Activity report management command
+"""
+
+# Imports #####################################################################
+
+import argparse
+from configparser import ConfigParser
+import csv
+from datetime import datetime, timedelta
+import os
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from instance import ansible
+from instance.models.openedx_instance import OpenEdXInstance
+from registration.models import BetaTestApplication
+
+
+def valid_date(s):
```

Comment:
@pcockwell, nit: missing docstring.

Reply:
Resolved in next commit

Reply:
@pcockwell, can you push the commit(s) made for addressing the review comments and fixing the CI errors?

Reply:
Done